serverjs/updatecards.js

```diff
@@ -463,10 +539,10 @@ var specialCaseCardsList = {
     tokenId: "e6fa7d35-9a7a-40fc-9b97-b479fc157ab0"
   }],
   "Westvale Abbey": [{
-    tokenId: " 94ed2eca-1579-411d-af6f-c7359c65de30",
+    tokenId: "94ed2eca-1579-411d-af6f-c7359c65de30",
```

Comment:
Is there any way we can move this object to a data file i.e. json rather than in the source code?